nvflare/private/fed/simulator/simulator_app_runner.py

```diff
@@ -40,6 +40,9 @@ def create_run_manager(self, args, conf, federated_client, workspace):
 
 
 class SimulatorServerAppRunner(ServerAppRunner):
+    def __init__(self, server) -> None:
+        super().__init__(server)
+
```

Comment:
if `SimulatorServerAppRunner` share the same init method as parent class, maybe we don't need this part.

Reply:
This maybe not necessary. Does not hurt.